### PR TITLE
Add resource printer and describer for PodDisruptionBudget

### PR DESCRIPTION
**Pull request by soltysh**
@davidopp || @mml ptal

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/35287)

### CODE REVIEW

**Comment by davidopp** [comment]
@mwielgus

File: pkg/kubectl/describe.go [hunk 1]
```diff
@@ -2385,6 +2387,36 @@ func (s *StorageClassDescriber) Describe(namespace, name string, describerSettin
 	})
 }
 
+type PodDisruptionBudgetDescriber struct {
+	clientset.Interface
+}
+
+func (p *PodDisruptionBudgetDescriber) Describe(namespace, name string, describerSettings DescriberSettings) (string, error) {
```
Comment (mwielgus) [inline pkg/kubectl/describe.go]: I guess we should print some status information as well. However I have a pr in flight that changes the status struct a bit #34546. The PR should be merged on Monday.

Comment (soltysh) [inline pkg/kubectl/describe.go]: I'll wait for the other PR to land and will update this accordingly, in that case.

**Comment by mwielgus** [comment]
We need this PR back in shape ASAP. I completely forgot about it. If you don't have time to work on this would it be ok for me to cherry-pick your commit and add some fixes on top of it?

**Comment by soltysh** [comment]
Superseded by #36495.

**Pull request closed without merging**